core-common/src/test/java/org/mapstruct/test/PackagePrivateMapperTest.java

```diff
@@ -0,0 +1,33 @@
+/**
+ *  Copyright 2012-2017 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.test;
+
+import org.junit.Test;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+public class PackagePrivateMapperTest {
```

Comment:
Could you omit the doubled new lines, add a line break at EOF and a `@IssueKey` annotation?

Reply:
I don't know this `@IssueKey` annotation. Can you point me to the doc ?

Reply:
The problem is that `@IssueKey` is defined into the _mapstruct-processor_ module which is not accessible from _mapstruct-common_. Should I add a dependency with the _test_ scope ?

Reply:
I have tried to package the test classes from _mapstruct-processor_ and add a _test_ dependency on it from _mapstruct-common_. However I have the feeling that this is a strange dependency. In my opinion the `@IssueKey` must be moved into _mapstruct-common\src\test_  and used in other modules by adding a dependency to _mapstruct-common_ instead.
What do you think/should I do ?

Reply:
Hi, agreed, this dependency isn't good. When adding the original comment, I didn't notice that your change is in the _common_ module and thus can't access the `@IssueKey` annotation. I'd suggest to just omit it then, seems not worth to move this. Sorry for the confusion.